uaclient/entitlements/esm.py

```diff
@@ -26,6 +26,7 @@ class ESMInfraEntitlement(ESMBaseEntitlement):
     title = "ESM Infra"
     description = "UA Infra: Extended Security Maintenance"
     repo_key_file = "ubuntu-advantage-esm-infra-trusty.gpg"
+    help_info = "esm-infra help"
```

Comment:
Can we load this from a .yaml file or something that we load this from so that we can give a single file of content for the list of features to product management and have them submit something back easily?